CSharp_Net_module1_7_1_lab/CSharp_Net_module1_7_1_lab/Computer.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Net_module1_7_1_lab
+{
+    delegate int del(int i);
```

Comment:
I think this delegate is useless

Reply:
o... Really, I forgot to clear trash... This progect was opened when I passed TEST, and I checed some code here.... And didn`t clear all. Thanks) Fixed i 7cf26dfcf12410763605a17ee989cb96e47a5308